crates/bevy_render/src/camera/projection.rs

```diff
@@ -197,14 +197,16 @@ pub enum ScalingMode {
 ///
 /// Note that the scale of the projection and the apparent size of objects are inversely proportional.
 /// As the size of the projection increases, the size of objects decreases.
+///
+/// Note also that the view frustum is centered at the origin.
```

Comment:
However a justification here would be useful, like "This helps in 2D when both the camera and sprites are likely to be placed at z=0.0"